lib/session.js

```diff
@@ -55,8 +55,8 @@ module.exports = class Session {
     return cookieSignature.sign(this.sessionId, this[secretKey])
   }
 
-  static restore (idGenerator, cookieOpts, secret, prevSession) {
-    const restoredSession = new Session(idGenerator, cookieOpts, secret, prevSession)
+  static restore (request, idGenerator, cookieOpts, secret, prevSession) {
```

Comment:
Is this change tested?

Reply:
Restore is only referenced once; To be exact in fastifySession line 80 (https://github.com/fastify/session/pull/64/files#diff-c327147bd50b6dd851469add08b782e4ede4da6003ddbab99f46e50ff220d84bR80-R84)

I don't see how this could be tested? I only added a request parameter to it which will be called inside Session's constructor which is tested in theory. Nevertheless I will add a rolling false custom id generator test case